data_collection/other_integrations/web_log_analytics.rst

```diff
@@ -101,3 +105,29 @@ Supported log formats:
 - log files of some popular Cloud Saas services: Amazon CloudFront logs, Amazon S3 logs
 - streaming media server log files such as: Icecast
 - log files with and without the virtual host will be imported
+
+The script does not directly support importing logs from log aggregator tools, like Grafana Loki or ELK. If you'd like to import logs from one of those, you need to download them to the disk first.
+
+
+Performance considerations & rate limiting
+----------------------
+
+The script needs CPU to read and parse the log files, but it is usually Piwik PRO server itself which will limit the import speed due to network latency.
+To improve performance, you can use the **--recorders** option to specify the number of parallel threads which will import hits into Piwik PRO. We recommend to increase **--recorders=N** until import speed is satisfying for you.
```

Comment:
I have sincere doubts about that question. Default settings allow to reach speeds over 1.5k req/sec (most of the time). I'd say that "we recommend not to touch the default values until you really need to" :)
If we really need to suggest any parameter value tweaks then my recommendation would be to keep the `--recorders` at max 2 and to play with `--recorder-max-payload-size` instead (untill they reach `max payload size` http error from the CollectorFront).

Reply:
We disabled bulk tracking by default and by default it will run with `--recorders=1`, sow very very slow. So instead of recommending how to tweak recording speed I would just describe generally, how parameters that relates to recording speed works.